src/main/java/uk/gov/register/util/DateFormatChecker.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.register.util;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.time.DateUtils;
+
+import java.text.ParseException;
+
+import static uk.gov.register.util.DateFormat.ISO_8601.*;
+
+public class DateFormatChecker {
+
+    public static boolean isDateFormatValid(final String dateToValidate) {
+        if (StringUtils.isEmpty(dateToValidate)) {
+            return true;
+        }
+
+        try {
+            DateUtils.parseDate(dateToValidate, YEAR, YEAR_MONTH, YEAR_MONTH_DAY, YEARS_MONTH_DATA_TIME, YEARS_MONTH_DATA_TIME_UTC);
```

Comment:
Should we use [parseDateStrictly](https://commons.apache.org/proper/commons-lang/javadocs/api-3.1/org/apache/commons/lang3/time/DateUtils.html)?